Dockerfile

```diff
@@ -1,10 +1,9 @@
-FROM google/dart-runtime-base:1.25.0-dev.9.0
+FROM google/dart-runtime-base:2.0.0-dev.8.0
 
- # After install we remove the apt-index again to keep the docker image diff small.
- RUN apt-get update && \
-     apt-get upgrade -y && \
-     apt-get install -y git unzip && \
-     rm -rf /var/lib/apt/lists/*
+# After install we remove the apt-index again to keep the docker image diff small.
+RUN apt-get update && \
```

Comment:
why remove upgrade?

Reply:
Upgrade upgrades **everything* that's installed, including `Dart`.

We're set up to use a recent Dart image – which uses a recent Google Cloud image. So I think we're good!